acapi2/resources/environment.py

```diff
@@ -313,5 +313,13 @@ def update_log_forwarding_destinations(
             "address": address,
         }
         response = self.request(uri=uri, method="PUT", data=data)
-
         return response
+
+    def disable_job(self, cron_id: str) -> dict:
+        """
+        Disables a cron job.
+        :cron_id: Identifier for cron job
+        """
+        uri = f"{self.uri}/crons/{cron_id}/actions/disable"
+        response = self.request(uri=uri, method="POST", data="")
+        return response.json()
```

Comment:
From now on I'll start enforcing the return of the Response object itself, it's practical to receive only the json object, but it could be a pain to debug if something goes wrong with the request. 

Returning the response itself gives more control to whoever uses this endpoint.

Reply:
@pmatias okay will update that one.

Reply:
@pmatias I have updated it .Please check